src/api.rs

```diff
@@ -175,12 +175,13 @@ impl QuicP2p {
     ///
     ///     config.local_port = Some(3001);
     ///     let mut quic_p2p = QuicP2p::with_config(Some(config), &[peer_addr], true)?;
-    ///     let endpoint = quic_p2p.bootstrap().await?;
+    ///     let endpoint = quic_p2p.bootstrap(None).await?;
     ///     Ok(())
     /// }
     /// ```
     pub async fn bootstrap(
         &self,
+        override_target_addresses: Option<&Vec<SocketAddr>>,
```

Comment:
I don't think we need the `Option` just a slice `&[SocketAddr]`, it's not empty you override.